packages/server/src/next-utils.ts

```diff
@@ -162,7 +162,7 @@ export async function nextStart(nextBin: string, buildFolder: string, config: Se
       rej("")
     } else {
       const nextjs = spawn(nextBin, spawnCommand, {
-        cwd: process.cwd(),
+        cwd: buildFolder,
```

Comment:
Base path fix looks great!

Was this change needed?

Reply:
@flybayer, yes it seem so.  This will make blitz build + blitz start to serve static assets correctly.  Without this change, blitz start will serve next assets from _next/ instead of ${adminPath}/_next/, while API path is served from ${adminPath}/api and root is served from ${adminPath}/.

<img width="865" alt="Screen Shot 2021-03-08 at 3 45 11 PM" src="https://user-images.githubusercontent.com/14841421/110396912-ad9fac00-8025-11eb-9924-a663c4ebd229.png">
<img width="893" alt="Screen Shot 2021-03-08 at 3 47 38 PM" src="https://user-images.githubusercontent.com/14841421/110396918-b09a9c80-8025-11eb-8f7d-6a60189d6a46.png">

Reply:
ok yes, makes sense! Thanks!